test/functional/http/post/transactions.js

```diff
@@ -43,4 +56,24 @@ describe('POST /api/transactions (general)', () => {
 				);
 			});
 	});
+
+	describe('transaction processing', () => {
+		sendTransactionPromise(transaction).then(res => {
+			expect(res.body.data.message).to.be.equal('Transaction(s) accepted');
+		});
+		return phases.confirmation([transaction], []);
+	});
+
+	describe('validation', () => {
+		it('should fail when an already confirmed transaction sent again', () => {
```

Comment:
not best description, maybe `should fail when trying to send already confirmed transaction / transaction that is already confirmed`